### DISCUSSION TITLE
Script is located `priv/tracker/js/plausible.js` but accessible at `js/plausible.js`. How?

### DISCUSSION THREAD

**Discussion by ally9335** [plausible/analytics#2138] [Anything Else]
The tracking  script `plausible.js` is located in `priv/tracker/js/plausible.js`

But it somehow is accessible at `https://plausible.io/js/plausible.js`, that is, as if it was not at `priv/tracker/js/plausible.js` but simply at `priv/js/plausible.js`. But it's not located at `priv/js/plausible.js` even after all of the assets have been compiled!

How can this work?

**Comment by RobertJoonas** [comment]
Hi @ally9335! The tracker files are served by this Elixir module: https://github.com/plausible/analytics/blob/master/lib/plausible_web/plugs/tracker.ex. Hope this helps!
